app/stacks/types.ts

```diff
@@ -98,7 +98,14 @@ export type ChatsStackParamList = {
 		title: string;
 		data: IOptionsField[];
 		value?: any; // TODO: Change
-		onChangeText?: ((text: string) => IOptionsField[]) | ((term?: string) => Promise<any>);
+		onChangeText?: (text?: string) => Promise<{ data: IOptionsField[] } | undefined>;
+		onEndReached?: (
+			text: string,
+			offset?: number
+		) => Promise<{ data: IOptionsField[]; total: number; offset: number } | undefined>;
```

Comment:
Since we're adding this, we should fix this return of `IOptionsField`.
That's coming from `NotificationPreferencesView`.
PickerView is supposed to be generic.